abracadabra/src/geometries/sipm_hamamatsu_blue-test.cc

```diff
@@ -105,6 +106,9 @@ TEST_CASE("hamamatsu app", "[app]") {
     run_manager -> BeamOn(1);
   }
 
+  // Add persistency manager
+  auto persistency = new persistency_manager{};
```

Comment:
The name isn't used anywhere (will it ever be?) so remove the name. The comment should probably say something like "G4 automatically takes charge of lifetime of persistency manager".